back/mail-processing/package.json

```diff
@@ -7,7 +7,8 @@
     "sls:deploy-dev": "env-cmd -f ./conf/dev.env sls deploy",
     "sls:deploy-prod": "env-cmd -f ./conf/prod.env sls deploy",
     "sls:deploy-ci": "serverless deploy",
-    "run-dev": "env-cmd -f ./conf/dev.env ./run.py"
+    "run-dev": "env-cmd -f ./conf/dev.env ./run.py",
+    "compile-emails": "node mail-mjml/compile-emails.js"
```

Comment:
should compile emails command be executed before deploy on CI ?

Reply:
Yes, they can be executed before deploy automatically

Reply:
can or should ? :)

Reply:
It depends on the point of view...